angular-ngrx-scss/src/app/repository/components/repo-header/repo-navigation/repo-navigation.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-repo-navigation',
+  templateUrl: './repo-navigation.component.html',
+  styleUrls: ['./repo-navigation.component.scss'],
+})
+export class RepositoryNavigationComponent {
+  @Input() basePath = '';
+  @Input() issuesCount = 123;
```

Comment:
Same comment as Linda left above - probably makes sense to make the default values 0 later.